### PR TITLE
[15.0][MIG] hr_study: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT42005

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1:

File: hr_study/models/hr_study.py [hunk 1]
```diff
@@ -10,5 +10,5 @@ class HrStudy(models.Model):
     _order = "sequence, id"
 
     sequence = fields.Integer(default=10)
-    name = fields.Char(string="Name", required=True, tracking=True)
+    name = fields.Char(required=True)
```
Comment (pedrobaeza) [inline hr_study/models/hr_study.py:13]: Why removing the tracking?

Comment (victoralmau) [reply hr_study/models/hr_study.py:13]: Because this model does not have `mail.thread` inheritance.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**